Library/Homebrew/formula.rb

```diff
@@ -405,11 +413,19 @@ def linked_keg
     Pathname.new("#{HOMEBREW_LIBRARY}/LinkedKegs/#{name}")
   end
 
+  def latest_head_prefix
+    head_prefixes = installed_prefixes.select do |pn|
+      pn.basename.to_s.start_with?("HEAD")
```

Comment:
I also noticed this. I think we need to add "Tab stores stable/devel/HEAD" logic to this PR, because it's getting difficult to keep in mind both this branch and the other PR branch in mind. We began to discuss `head_prefix_up_to_date?` here and there is no such method at all on this branch. Or we should carefully think what we merge in this PR, which seems to be more difficult.